packages/sync-actions/src/order-actions.js

```diff
@@ -146,42 +146,36 @@ export function actionsMapReturnsInfo(diff, oldObj, newObj) {
       }
       return []
     },
-    [CHANGE_ACTIONS]: (oldSReturnInfo, newReturnInfo) => {
-      const updateActions = Object.keys(returnInfoDiff).reduce(
```

Comment:
The way `createBuildArrayActions` works is that it actually returns the `diffKey` from which you have to extract the changes.

This part was removed as we were extracting the changes from all the items in each iteration of `createBuildArrayActions`, which is not needed and it was causing an error when all the items of 2 returnInfo have changed.

That is basically the reason why the signature now is `[CHANGE_ACTIONS]: (oldSReturnInfo, newReturnInfo, key)`,
where `key` was previously obtained from iterating over `Object.keys(returnInfoDiff)` 👍